membership-attribute-service/app/controllers/AccountController.scala

```diff
@@ -241,7 +193,7 @@ class AccountController(
       def getPaymentMethod(id: PaymentMethodId) = tp.zuoraRestService.getPaymentMethod(id.get).map(_.toEither)
       val maybeUserId = request.user.map(_.id)
 
-      logger.info(s"Attempting to retrieve payment details for identity user: ${maybeUserId.mkString}")
+      logger.info(s"Deprecated function called: Attempting to retrieve payment details for identity user: ${maybeUserId.mkString}")
```

Comment:
Just putting this log message in so we can see if this deprecated method is still called